tools-public/toolpad.yaml

```
@@ -112,5 +119,869 @@ nodes:
+    parentProp: children
+    parentIndex: a0
+  y623ild:
+    name: codeComponent_HealthBadge
+    props:
+      warning:
+        type: const
+        value: 10
+      problem:
+        type: const
+        value: 15
+      lowerIsBetter:
+        type: const
+        value: true
+      value:
+        type: jsExpression
+        value: |
+          Math.round(getRatio.data.ratio * 10000) / 100
+    attributes:
+      component:
+        type: const
+        value: codeComponent.HealthBadge
+    layout: {}
+    id: y623ild
+    type: element
+    parentId: e333iuo
+    parentProp: children
+    parentIndex: a0
+  jb63icx:
+    name: PageRow1
+    props: {}
+    attributes:
+      component:
+        type: const
+        value: PageRow
+    layout: {}
+    id: jb63icx
+    type: element
+    parentId: 8n03iat
+    parentProp: children
+    parentIndex: a1
+  rg53iur:
+    name: Text
+    props:
+      value:
+        type: const
+        value: Based on last 30 days invoices
+    attributes:
+      component:
+        type: const
+        value: Text
+    layout: {}
+    id: rg53iur
+    type: element
+    parentId: jb63icx
+    parentProp: children
+    parentIndex: a0
+  fn03hvq:
+    name: cICompletionTime
+    attributes:
+      title:
+        type: const
+        value: cICompletionTime
+      parameters:
+        type: const
+        value:
+          - - repository
+            - mui-x
+    id: fn03hvq
+    type: page
+    parentId: 9q8eshxx
+    parentProp: pages
+    parentIndex: a3
+  ii53hyb:
+    attributes:
+      query:
+        type: const
+        value:
+          method: GET
+          headers: []
+          browser: false
+          url:
+            type: jsExpression
+            value: >
+              `https://circleci.com/api/v2/insights/github/mui/${parameters.repository}/workflows/pipeline/summary?analytics-segmentation=web-ui-insights&reporting-window=last-7-days&workflow-name=pipeline`
+      connectionId:
+        type: const
+        value: null
+      dataSource:
+        type: const
+        value: rest
+    name: queryInsights
+    id: ii53hyb
+    type: query
+    parentId: fn03hvq
+    parentProp: queries
+    parentIndex: a0
+    params:
+      - - repository
+        - type: jsExpression
+          value: |
+            page.parameters.repository
+  3x93hxi:
+    name: PageRow
+    props: {}
+    attributes:
+      component:
+        type: const
+        value: PageRow
+    layout: {}
+    id: 3x93hxi
+    type: element
+    parentId: fn03hvq
+    parentProp: children
+    parentIndex: a0
+  ey83hg3:
+    name: codeComponent_HealthBadge
+    props:
+      value:
+        type: jsExpression
+        value: >
+          Math.round((queryInsights.data.metrics.duration_metrics.median / 60) *
+          100) /
+            100
+      warning:
+        type: const
+        value: 15
+      problem:
+        type: const
+        value: 20
+      unit:
+        type: const
+        value: minutes
```

I think that there should be a space here.

```suggestion
        value: ' minutes'
```